core/core.cc

```diff
@@ -359,6 +388,8 @@ static TaskGraph default_graph()
   graph.kernel = {KernelType::EMPTY, 0};
   graph.output_bytes_per_task = sizeof(std::pair<long, long>);
 
+  graph.kernel.scratch_bytes_per_task = 0;
```

Comment:
Can we store scratch_bytes_per_task on `graph`, like output_bytes_per_task?

Reply:
Store `scratch_bytes_per_task` in `graph`,
but still keep `scratch_bytes_per_task` in `kernel`, since `void execute_kernel_memory` need this parameter